source/contribute.rst

```diff
@@ -57,31 +57,42 @@ Initial Setup
     $ cd orm
     $ git remote add doctrine git://github.com/doctrine/orm.git
 
--  Adjust your branch to track the doctrine master remote branch, by
-   default it'll track your origin remote's master:
+Branching from the default branch
+---------------------------------
 
-.. code-block:: console
+New pull requests are created with the repository's default branch as base branch.
+The default branch is the branch you see when you enter the repository page on GitHub.
+
+.. image:: ../images/default-branch.png
+   :alt: The default branch
+   :style: margin-bottom: 20px
 
-    $ git config branch.master.remote doctrine
+In this ORM example, it's the branch with the name **2.7**. The branch name reflects the
```

Comment:
Maybe the ORM is not the best example… naming a branch without a `.x` is kinda deprecated since the automatic releases workflow creates them with that suffix. Consider using the DBAL as an example.

Reply:
Good point. I've changed it to DBAL.